### ISSUE TITLE
Modal not showing data - only undefined

### ISSUE THREAD

**Issue by webnoob** [wffranco/vue-strap#34]
I have the following:

```
<modal title="Make Payment" v-model="modelProp.value">
  <div slot="modal-header" class="modal-header">
    <h4 class="modal-title">Make a payment</h4>
  </div>
  <div>
    <p>Make a payment</p>
    <input type="text" v-model="paidAmount"/>
  </div>
</modal>
```

And when I click my button to trigger the modal, I get the modal itself show and the background fade out however, the content shows "undefined, undefined, undefined". This was working a few days ago and the only thing I can see that has changed is that I've done a fresh `npm install` so have the latest build now.

This is what shows:

![image](https://cloud.githubusercontent.com/assets/10404747/21580615/5db49f94-cffc-11e6-8146-524c9bb7480b.png)

This the element source:

```
<div role="dialog" class="modal in" style="display: block;">
  <div role="document" class="modal-dialog">
    <div class="modal-content">undefined undefined undefined</div>
  </div>
</div>
```

Any clues?

**Comment by webnoob** [comment]
I'v just noticed from this thread: https://github.com/wffranco/vue-strap/issues/31 that it's been advised to update to the later version of vue. I'm trying this now to see if it helps my issue.

**Comment by gholol** [comment]
@webnoob Yeah, same thing happens with aside and input components. The whole release is broken.

**Comment by webnoob** [comment]
@gholol Thanks for that, it appears to have fixed the issue.

**Comment by dontfeedthecode** [comment]
@webnoob @gholol FYI I've just updated to 2.1.9 and the problem persists.

**Comment by webnoob** [comment]
Are we sure this is a problem with Vue and not with vue-strap in that case? Has something changed in the latest vue version which requires changes within vue-strap?

**Comment by poppabear8883** [comment]
I can confirm this issue is still persistent in 2.1.10 ... #41

**Comment by gholol** [comment]
@webnoob 

>Are we sure this is a problem with Vue and not with vue-strap in that case?

Yes because it works as soon as you downgrade to 2.1.6

**Comment by webnoob** [comment]
That just means that the code in vue-strap works with the code in vue 2.1.6 not that vue has changed something _that it's not intending to fix_ and vue-strap also needs to adhere to that change.

**Comment by poppabear8883** [comment]
@gholol @webnoob see #43 #41 .. needs a new build

**Comment by wffranco** [comment]
last build with vue 2.1.10

**Comment by stefanoborini** [comment]
Still present in vue 2.3.2
